packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -70,7 +70,17 @@ interface ProcessTelemetryProperties {
 }
 
 /**
- * Base class from which all shared objects derive.
+ * Base class from which all {@link ISharedObject|shared objects} derive.
```

Comment:
I'm interested in others' thoughts on this (@ChumpChief @tylerbutler in particular), but is "Shared Object" a formal Fluid concept? If so, should we be capitalizing it where we reference it? Our repo has a mix of capitalized and uncapitalized, and it would be nice to have consistent guidance.

In my opinion, we are referring to a specific Fluid concept in these cases, so it _should_ be capitalized.

Reply:
I think it could probably be retired over time if some of the referenced interface cleanup happens.  I personally tend to use "DDS" now wherever I might have said Shared Object before.